application/src/main/java/com/sanction/lightning/twitter/TwitterService.java

```diff
@@ -79,6 +79,8 @@ public TwitterUser getTwitterUser() {
    *
    * @return The URL to redirect to for authentication or {@code null} if unable to fetch the URL.
    */
+
+  // TODO Finish this method
```

Comment:
Can you move this TODO to above the return statement and say "TODO: replace redirect URL" or something so that it's more specific?